### ISSUE TITLE
Duik IK stop working after reopen the project.

### ISSUE THREAD

**Issue by Jackredgrave** [RxLaboratory/DuAEF_Duik#880]
After I open the project file, the IK controllers wont move the body part anymore but controllers are still there moving.

Result in this [clip](https://drive.google.com/file/d/1ql6J5PJ4SdzHfvaWgejozg5Neg2kF85j/view?usp=sharing)

**Comment by Nico-Duduf** [comment]
Hi,

What do you expect I can do without more details?

What version of Ae is this, what OS, what version of Duik?
How was the character rigged? using the puppet tool?
Have you checked for expression errors?
Have you checked the parenting?
Have you checked the puppet effects?
etc.

**Comment by Jackredgrave** [comment]
- After Effects version 17.0.0
- Duik version 16.2.21
- I'm not using puppet tool. I used IK auto-rig with structures on separate layers. (following this [tutorial](https://www.youtube.com/watch?v=9De7K0-ujOo&t=1156s))
- No expression errors found.
- I checked the parenting and it is the same as the first time I save and close the project. (artwork layers -> structure layers, thigh -> S l thigh, calf -> S l calf, etc.) It seem like the controllers layers no longer work as a controller anymore.

Luckily, the old controllers layers still have their keyframes so I created a new controller with auto-rig and paste the keyframe.
